appmonitoring/ts/src/CertificateGenerator.ts

```diff
@@ -232,6 +233,53 @@ export class CertificateManager {
         return caCertificate.verify(certificate);
     }
 
+    private static IsValidCertificate(operationId: string, mwhcCaBundle: string, webhookCertData: WebhookCertData, clusterArmId: string, clusterArmRegion: string): boolean {
+        try {
+            forge.pki.certificateFromPem(mwhcCaBundle);
+            forge.pki.certificateFromPem(webhookCertData.caCert);
+            forge.pki.certificateFromPem(webhookCertData.tlsCert);
+            forge.pki.privateKeyFromPem(webhookCertData.tlsKey);
+            return true;
+        } catch (error) {
+            logger.error('Error occured while trying to validate certificates!', operationId, this.requestMetadata);
+            logger.error(JSON.stringify(error), operationId, this.requestMetadata);
```

Comment:
We want a single error trace, why split it into 2?

Reply:
I dont think this comes out as 2? I need that distinct message, and not have it be muddled with the stacktrace

Reply:
You call logger.error twice, how would it not come out as 2? You can do it this way if that's what you want, but the output will be just as muddled, except now there are two error messages that pertain to exactly the same thing.